CiE-iOS/CiE-iOS/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LoginViewController.swift
+//  CiE-iOS
+//
+//  Created by Jack Tan on 20/5/18.
+//  Copyright © 2018 Nikita Hans. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    @IBOutlet weak var usernameField: UITextField!
+    @IBOutlet weak var passwordField: UITextField!
+    
+    @IBAction func LoginButtonPressed(_ sender: Any) {
```

Comment:
please write functions with none capital letters.
try to link functions to UIElements via the ViewController not the storyboard this will make later changes more easy and readable.